docs/development/workflow/maintainer_workflow.rst

```diff
@@ -142,25 +134,20 @@ followed by IPython:
   * The next X.Y release +1; for example if 0.3 is the next release, there should also be a milestone for 0.4 for
     issues that are important, but that we know won't be resolved in the next release.
 
-  * Future--this is for all issues that require attention at some point but for which no immediate solution is in
-    sight.
-
-* Bug fix release milestones should only be used for deferring issues that won't be fixed in the next minor release,
-  or for issues is previous releases that no longer apply to the mainline.
-
-* When in doubt about which milestone to use for an issue, use the next minor release--it can always be moved once
-  it's been more closely reviewed prior to release.
-
 * Active milestones associated with a specific release (eg. v0.3.0) should contain at least one issue with the
```

Comment:
I don't get this point. Github doesn't autoclose milestones (any more?). Also, I think we don't have "release label"

Reply:
Should I just delete this whole point? I didn't get it either but I don't understand enough about the release nitty gritty on this repo to touch it.

Reply:
👍 on deleting, I think it is very obsolete.

Reply:
I did keep the "rolling reminder" as it is still relevant. Does it look better now?